### PR TITLE
Introduce token based authentication

### PR DESCRIPTION
**Pull request by schmijos**

### CODE REVIEW

**Comment by dbrgn** [comment]
Cool. Do you have a quick summary for the README?

File: test/models/api_consumer_test.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+require 'test_helper'
+
+class ApiConsumerTest < ActiveSupport::TestCase
+  # test "the truth" do
+  #   assert true
+  # end
```
Comment (dbrgn) [inline test/models/api_consumer_test.rb]: This should probably be completed or removed.

Comment (schmijos) [inline test/models/api_consumer_test.rb]: Good point.

**Comment by schmijos** [comment]
I also added a paragraph about token auth to the README

**Comment by dbrgn** [comment]
Cool, thanks.

**Comment by dbrgn** [comment]
I think it looks good now, not that I'm good at reading Ruby though :)

**Pull request merged**